cron.go

```diff
@@ -71,13 +72,19 @@ func (s byTime) Less(i, j int) bool {
 
 // New returns a new Cron job runner.
```

Comment:
A small update would probably be helpful:

// New returns a new Cron job runner, in the Local time zone.